curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64df202aa1342114cd077920.md

```diff
@@ -0,0 +1,92 @@
+---
+id: 64df202aa1342114cd077920
+title: Step 39
+challengeType: 20
+dashedName: step-39
+---
+
+# --description--
+
+If the number of disks is even and the remainder equals `1`, the move is allowed between the source rod and the auxiliary rod. Add an `else` clause to print the allowed movement and call `make_allowed_move` with the correct arguments.
+
+If you look at the output, you can see that the execution stops at the third move because of an `IndexError`. This happens because the code is still incomplete and needs an `else` clause you will be writing soon. To make it work, turn your `make_allowed_move()` call into a comment.
```

Comment:
```suggestion
If you look at the output, you can see that the execution stops at the third move because of an `IndexError`. This happens because the code is still incomplete and needs an `else` clause that you will be writing soon. To make it work, turn your `make_allowed_move()` call into a comment.
```